Source/WebCore/css/CSSValue.h

```diff
@@ -217,91 +217,89 @@ class CSSValue {
 protected:
     static const size_t ClassTypeBits = 7;
 
-    // FIXME: Use an enum class here so we don't have to repeat "Class" in every name.
-    enum ClassType {
-        PrimitiveClass,
+    enum class ClassType : uint32_t {
```

Comment:
It's not very important because we hardcode the actual field size to be 7 bits (`ClassTypeBits`), but this could be declared as a `uint8_t` if it's used somewhere else.

Reply:
Done.